<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Focus</class>
 <widget class="QDialog" name="X2Focus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>404</width>
    <height>628</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>404</width>
    <height>628</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EFA settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>99999</width>
       <height>99999</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>8</y>
        <width>360</width>
        <height>536</height>
       </rect>
      </property>
      <property name="title">
       <string>EFA settings</string>
      </property>
      <widget class="QCheckBox" name="isFanOn">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>328</y>
         <width>85</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Fan On</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="isStopDetect">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>360</y>
         <width>240</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Detect and stop if hitting something</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>256</y>
         <width>230</width>
         <height>60</height>
        </rect>
       </property>
       <property name="title">
        <string>Goto approach direction</string>
       </property>
       <widget class="QRadioButton" name="radioButtonAppPos">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>70</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Positive</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonAppNeg">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>80</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Negative</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>392</y>
         <width>300</width>
         <height>128</height>
        </rect>
       </property>
       <property name="title">
        <string>Temperatures</string>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>180</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Source to report for focuser :</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>70</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Ambient :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>80</y>
          <width>70</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Primary :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>80</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Secondary :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="tempSource">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>30</y>
          <width>90</width>
          <height>24</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Primary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ambient</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Secondary</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="A_Temp">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>56</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>A_Temp</string>
        </property>
       </widget>
       <widget class="QLabel" name="P_Temp">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>56</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>P_Temp</string>
        </property>
       </widget>
       <widget class="QLabel" name="S_Temp">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>60</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>S_Temp</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>32</y>
         <width>328</width>
         <height>208</height>
        </rect>
       </property>
       <property name="title">
        <string>Position and limits</string>
       </property>
       <widget class="QLabel" name="maxPos">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>88</y>
          <width>72</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>00000000</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>24</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>New Position :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>56</y>
          <width>185</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibrate limits</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="newPos">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>24</y>
          <width>130</width>
          <height>24</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>128</y>
          <width>264</width>
          <height>64</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;After calibrating you need to disconnect and reconnect for TheSkyX to take the new values into account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>256</x>
          <y>24</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>16</x>
          <y>88</y>
          <width>91</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Position :</string>
        </property>
       </widget>
       <widget class="QLabel" name="isCalibrated">
        <property name="geometry">
         <rect>
          <x>208</x>
          <y>88</y>
          <width>112</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Not Calibrated</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>192</x>
        <y>560</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>560</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
